mender-convert-package

```diff
@@ -199,8 +199,10 @@ else
     log_fatal "Could not determine root file-system type. Aborting..."
 fi
 
-disk_create_file_system_from_folder "work/rootfs/data/" "work/data.img" \
-    "${data_part_sectors}" "${image_fs_type}"
+# NOTE: Since the data partition has a dependency on the rootfs.img checksum
+# when writing the bootstrap Artifact we have to move the datadir out of the
+# way, and write the rootfs image first.
+run_and_log_cmd "mv --force work/rootfs/data work/data"
 
 # Clear this area as it will be contained in the data.img
 sudo rm -rf work/rootfs/data/*
```

Comment:
Then we can remove these lines, right? I think we should to avoid confusion later on.

Reply:
nit pick: the removal of these lines should have been in the first commit